components/datetime/src/pattern/runtime/helpers.rs

```diff
@@ -16,10 +16,7 @@ use zerovec::ule::AsULE;
 /// item to be replaced allocating the `Pattern` only if needed.
 ///
 /// For a variant that replaces all matching instances, see `maybe_replace`.
-pub fn maybe_replace_first(
-    pattern: &mut runtime::Pattern,
```

Comment:
Question: Is this intentional to remove the runtime prefix here? I guess it's inside the `runtime` module.